XForm/XForm/Functions/String/Replace.cs

```diff
@@ -0,0 +1,142 @@
+﻿using Microsoft.CodeAnalysis.Elfie.Model.Strings;
+using System;
+using XForm.Commands;
+using XForm.Data;
+using XForm.Types;
+
+namespace XForm.Functions
+{
+    internal class ReplaceBuilder : IFunctionBuilder
+    {
+        public string Name => "Replace";
+        public string Usage => "Replace([InputColumnName], [FindText], [ReplaceText?], [ExactMatch?])";
```

Comment:
It doesn't look like ReplaceText is an optional argument below. I think it should be non-optional and users should type "" if they want replace with empty. (So keep the code as-is and take the ? out of the Usage. =)